### PR TITLE
Add calendar specifics

### PR DESCRIPTION
**Pull request by castrojo**

### CODE REVIEW

**Comment by castrojo** [comment]
Creating a SIG Calendar seems to have gotten more complex, these steps are from @roberthbailey when he created a new SIG Calendar. Just going to add them to the SIG creation process.

File: sig-governance.md [hunk 1]
```diff
@@ -24,14 +24,21 @@ In order to standardize Special Interest Group efforts, create maximum transpare
...
+* Request a new [kubernetes.slack.com](http://kubernetes.slack.com) channel (#sig-foo) from [@parispittman](https://github.com/parispittman) or [@castrojo](https://github.com/castrojo).  New users can join at [slack.kubernetes.io](http://slack.kubernetes.io).
 * Slack activity is archived at [kubernetes.slackarchive.io](http://kubernetes.slackarchive.io).  To start archiving a new channel invite the slackarchive bot to the channel via `/invite @slackarchive`
 * Organize video meetings as needed. No need to wait for the [Weekly Community Video Conference](community/README.md) to discuss. Please report summary of SIG activities there.
- * Request a Zoom account from from **@paris** or **@jorge**. You must set up a google group (see below) for the SIG leads so that all the SIG leads have the ability to reset the password if necessary.
+ * Request a Zoom account from from [@parispittman](https://github.com/parispittman) or [@castrojo](https://github.com/castrojo). You must set up a google group (see below) for the SIG leads so that all the SIG leads have the ability to reset the password if necessary.
```
Comment (roberthbailey) [inline sig-governance.md]: Still doesn't explain how to request it, just from whom. Can you put your slack handles and say to use that (or email) to make it explicit how to contact you?

File: sig-governance.md [hunk 2]
```diff
@@ -24,14 +24,21 @@ In order to standardize Special Interest Group efforts, create maximum transpare
...
  * Read [how to use YouTube](https://github.com/kubernetes/community/blob/master/community/K8sYoutubeCollaboration.md) for publishing your videos to the Kubernetes channel. 
  * Calendars
-   - Add the meeting to the community meeting calendar by inviting cgnt364vd8s86hr2phapfjc6uk@group.calendar.google.com.
-   - If you want a team calendar ensure that it is managed with the SIG team lead list you will create (see below). Leads and jobs can change over time and it is important that no SIG resources are tied to a single user's account. 
+   1. Create a calendar on your own account. Make it public.
+   2. Share it with all SIG leads with full ownership of the calendar -
+they can edit, rename, or even delete it.
```
Comment (roberthbailey) [inline sig-governance.md]: why is there a weird line break here?

**Comment by roberthbailey** [comment]
Thanks, please squash before merging.

**Pull request merged**